src/components/Collection.tsx

```diff
@@ -87,6 +87,12 @@ export default class Collection extends React.Component<CollectionProps, any> {
     );
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (this.props.isFetching && !nextProps.isFetching) {
```

Comment:
maybe we shouldn't scroll to the top if isFetching is set to false due to an error